src/components/Footer.vue

```diff
@@ -19,6 +19,9 @@
         />
       </svg>
     </a>
+    <a href="personvern">
```

Comment:
This is probably not the right way to link to other views, please let me know if there is a better way to do it.